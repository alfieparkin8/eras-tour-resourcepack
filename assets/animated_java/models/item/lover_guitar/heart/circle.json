{
	"parent": "animated_java:item/lover_guitar/circle",
	"textures": {
		"1": "animated_java:item/lover_guitar/heart_layer_1",
		"5": "animated_java:item/lover_guitar/heart_boots_layer_1"
	}
}