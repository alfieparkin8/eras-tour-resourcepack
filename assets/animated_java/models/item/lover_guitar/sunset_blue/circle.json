{
	"parent": "animated_java:item/lover_guitar/circle",
	"textures": {
		"2": "animated_java:item/lover_guitar/lover_blue"
	}
}