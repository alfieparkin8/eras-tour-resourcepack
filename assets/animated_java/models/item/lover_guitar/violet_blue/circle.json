{
	"parent": "animated_java:item/lover_guitar/circle",
	"textures": {
		"1": "animated_java:item/lover_guitar/violet_layer_1",
		"2": "animated_java:item/lover_guitar/lover_blue",
		"5": "animated_java:item/lover_guitar/violet_boots_layer_1"
	}
}