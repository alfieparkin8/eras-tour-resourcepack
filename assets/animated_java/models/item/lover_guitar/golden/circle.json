{"parent":"animated_java:item/lover_guitar/circle","textures":{"2":"minecraft:block/outfits/lover/golden_layer_1"}}